src/components/messages/ErrorMessage.tsx

```diff
@@ -13,6 +15,29 @@ type Props = {
   onTryAgain?: () => void
 }
 
+const StyledErrorIcon = styled(
+  ({
+    icon,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    color,
+    ...props
+  }: { icon: string; color?: Color } & React.ComponentProps<
+    typeof MaterialIcon
+  >) => <MaterialIcon {...props}>{icon}</MaterialIcon>,
+  {
+    shouldForwardProp: () => true,
+  },
+)(({ theme, color }) => ({
+  color: color ? theme.palette[color].main : undefined,
+}))
```

Comment:
Didn't have a good look at this, because I'm rushing to get it all done before the end of sprint review, but I think you can simplify this one.